wifiphisher.py

```diff
@@ -875,10 +877,29 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # if template is incomplete locally, delete and ask for a download
+    if not phishingpage.check_template(TEMPLATE_NAME):
```

Comment:
You should check if the template lies in `phishingpage.TEMPLATE_DATABASE` dictionary. If it doesn't the script should exit with a message as the user provided a non-existent template. If it does and the key value is `None` it means that we have it local, hence we can skip the `check_template` check.